packages/sqs/lib/sqs/AbstractSqsPublisher.ts

```diff
@@ -64,15 +72,12 @@ export abstract class AbstractSqsPublisher<MessagePayloadType extends object>
       }
 
       message = this.updateInternalProperties(message)
+      const maybeOffloadedPayloadMessage = await this.offloadMessagePayload(
```

Comment:
`offloadMessagePayloadIfNeeded` might be a more accurate name, as it is conditional